tests/specs/push.js

```diff
@@ -81,3 +80,23 @@ test('api.push(objects)', function (t) {
     })
   })
 })
+
+test('api.push(object)', function (t) {
+  t.plan(1)
+  var db = dbFactory('hoodieDB5')
+  var PouchDB = db.constructor
+  var remoteName = PouchDB.utils.uuid(10)
+  var api = db.hoodieSync({remote: remoteName})
+
+  var obj1 = {_id: 'test1', foo1: 'bar1'}
+  var obj2 = {_id: 'test2', foo1: 'bar2'}
+  var obj3 = {_id: 'test3', foo1: 'bar3'}
+  db.bulkDocs([obj1, obj2, obj3])
+
+  .then(function () {
+    api.push(obj1._id) // string
```

Comment:
ooops just saw that now. The test says `api.push(object)`. But what you really do is `api.push(id)`.

This line should be `api.push(obj1)`, not `api.push(obj1._id)` ... I'll fix